flowzone.yml

```diff
@@ -3184,7 +3184,8 @@ jobs:
 
   cloudformation_test:
     name: Test CloudFormation
-    runs-on: ${{ fromJSON(inputs.cloudformation_runs_on || inputs.runs_on) }}
+    # FIXME: https://github.com/actions/setup-python/issues/705
+    runs-on: ["Linux", "X64"] #${{ fromJSON(inputs.cloudformation_runs_on || inputs.runs_on) }}
```

Comment:
This is forcing self-hosted runners, which are not available to all repositories, especially outside of our org.

Also it doesn't address the other jobs that are using setup-python.

Reply:
I am tempted to leave those to the project maintainer(s).